storage/filestorage/filestorage.go

```diff
@@ -75,6 +79,7 @@ func (s *FileStorage) Get(_ *http.Request, image string) (*imagor.Blob, error) {
 
 // Put implements imagor.Storage interface
 func (s *FileStorage) Put(_ context.Context, image string, blob *imagor.Blob) (err error) {
+	defer prometheus.NewTimer(storage.OperationHistorgram.WithLabelValues("file", "put")).ObserveDuration()
```

Comment:
It does seem possible to use a `prometheus.ObserverFunc()` to have a "success" label (`err != nil`) (see https://github.com/prometheus/client_golang/blob/2771bcc5da1c3bde389e1905c360d830875567e1/prometheus/example_timer_complex_test.go#L47).

But in some cases the `error` returned is not declared at the top of the function, or it returns a function call, so while possible, it would be a bigger change than I'm comfortable with making at this point.